Amendment: Appendices (#8)
HackSoc/constitution: README.md

    1. HackSoc will be referred to as the 'Society'.
    2. The constitution of the Society, as laid out in [Constitution.md](./Constitution.md), will be referred to as the 'Constitution'.

## 2. General Rules

1. All changes to this repository MUST be made via a Pull Request.
2. 'Housekeeping' changes MAY be made on the conditions that said changes:
    1. MUST NOT affect the meaning of the Constitution or its appendices (though they MAY affect the format and/or style);

>     2. MUST be made via a Pull Request as per §2.1, for which the following restrictions apply:

Reviewer: for the sake of keeping things consistent with the rest of the constitution, at least for the time being:

```suggestion
    2. MUST be made via a Pull Request as per 2.1, for which the following restrictions apply:
```

(I would personally consider adding `§` to section numbers later to be a housekeeping change that would not require an AGM/EGM)

    2. MUST contain the results of any vote on the amendment they represent.
    3. SHOULD contain a summary of the discussion and debate of the proposed amendment they represent.
    4. MUST represent exactly one amendment to the Constitution.
2. Commit messages merging Pull Requests for amendments to the Constitution:
    1. MUST contain the results of the passing vote for the amendment they represent (i.e. the number of votes for, against, and abstaining).
    2. MUST start with the text 'Amendment:'.
3. Changes to the Constitution SHOULD be proposed by the creation of a Pull Request for an amendment to be discussed and voted on at the next General Meeting of the Society.
4. Ideas for changes to the Constitution MAY be proposed by the creation of an Issue, which MUST be refined into a specific change before it can be considered at a General Meeting.

## 4. Changes to Constitutional Appendices

1. Pull Requests for amendments to Constitutional Appendices:
    1. MUST be merged following a successful vote at a meeting of the Committee.
    2. MUST contain the result of any vote on the amendment they represent.

>     3. MUST represent an amendment to exactly one Appendix of the Constitution.

Reviewer: this (and 4.2.2) will make it difficult to merge, split, or update in bulk the amendments

>     3. MUST represent an amendment to exactly one Appendix of the Constitution.
>     4. MUST not affect the content of the Constitution.

Reviewer: given the possibility of merging/splitting appendices, and concerns raised about the word "affect":

```suggestion
    3. MUST not alter the content of the Constitution.
```

2. Commit messages merging Pull Requests for amendments to Constitutional appendices:
    1. MUST contain the results of the passing vote for the amendment they represent (i.e. the number of votes for, against, and abstaining)

>     2. MUST start with the text 'Appendix `X`:', where `X` is the letter of the affected appendix.

Reviewer: suggestion: add something along the lines of "or, if the amendment affects multiple appendices, 'Appendices:'"

Reviewer: ```suggestion
    2. MUST start with the text 'Appendix `X`:', where `X` is the letter of the affected appendix, if the commit affects exactly one appendix.
    3. MUST start with the text 'Appendices:' if the commit affects more than one appendix.
```
